website/src/content/main-concepts/proxy-to-external-api.mdx

```diff
@@ -16,74 +18,213 @@ Merchant Center applications can make [authenticated requests](/main-concepts/au
-The validation is performed by using the `/.well-known/jwks.json` endpoint exposed by the Merchant Center API Gateway. You can read more about JWKS [here](https://auth0.com/docs/jwks) and [here](https://tools.ietf.org/html/rfc7517).
+## Validating the JSON Web Token
 
-The endpoint exposes a public key which can be used to verify the JWT signature, according to the OIDC spec. We also expose a `/.well-known/openid-configuration` discovery endpoint.
+To facilitate the implementation of validating the JWT from the `Authorization: Bearer <token>` HTTP header, we provide a package with built-in functions and helpers to perform the heavy work.
 
-If your backend server is implemented with JavaScript, we recommend to use the [`express-jwt`](https://www.npmjs.com/package/express-jwt) package that acts as an Express middleware to automatically verify the authentication request.
+```console noPromptLines="2"
+npm install --save @commercetools-backend/express
 
-```js
-const express = require('express');
-const jwt = require('express-jwt');
-const jwksRsa = require('jwks-rsa');
+yarn add @commercetools-backend/express
+```
 
-// Initialize the app.
-const app = express();
+### Usage for Express.js
 
-// The issuer refers to the Merchant Center API Gateway URL.
-const issuer = 'https://mc-api.commercetools.com';
-// The audience refers to the forwarded target URL (same value as in `X-Forward-To` header)
-const audience = 'https://my-custom-app.com/foo/bar';
+The package includes an async Express.js middleware that performs the token validation and assigns a `session` object to the `request` object.
 
-app.use(
-  jwt({
-    // Dynamically provide a signing key based on the kid in the header
-    // and the singing keys provided by the JWKS endpoint
-    secret: jwksRsa.expressJwtSecret({
-      cache: true,
-      rateLimit: true,
-      jwksRequestsPerMinute: 5,
-      // The JWKS endpoint is also available from the discovery endpoint
-      // `${audience}/.well-known/openid-configuration`
-      // in case you wish to retrieve it programmatically.
-      jwksUri: `${issuer}/.well-known/jwks.json`,
-    }),
+```ts
+type TSession = {
+  userId: string;
+  projectKey: string;
+};
+```
+
+The middleware requires some options:
+
+- `audience` (_string_): The public-facing URL of your API server. The value should only contain the origin URL (protocol, hostname, port), the request path is inferred from the incoming request.
+
+  For example, given the external API is hosted at `https://my-api-server.com`, the `audience` value must be set to `https://my-api-server.com`.
+
+- `issuer` (_string_): Either a cloud identifier or a valid URL to the Merchant Center API Gateway. The cloud identifier maps to the Merchant Center API URL of the related [cloud region](/main-concepts/api-gateway#cloud-regions).
+
+  | Cloud Identifier | Merchant Center API Gateway URL |
```

Comment:
Nit: format the table to match rows and content maybe.

Reply:
I would prefer it like this, otherwise it gets messy when looking at diffs.